source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```
@@ -14,3 +14,72 @@
 .wporg-filtered-search-form {
 	display: block;
 }
+
+.wp-block-search.wp-block-search__button-inside .wp-block-search__inside-wrapper {
+	--wp--custom--button--hover--color--text: var(--wp--preset--color--charcoal-1);
+	--wp--custom--button--hover--color--background: var(--wp--preset--color--light-grey-2);
+	padding: 0;
+	border: none;
+	border-radius: var(--wp--custom--button--border--radius);
+	background: var(--wp--preset--color--white);
+	color: var(--wp--preset--color--charcoal-1);
+
+	.wp-block-search__input {
+		padding-right: 0;
+		border: none;
+	}
+
+	.wp-block-search__button {
+		padding-top: calc(var(--wp--preset--spacing--20) * 0.5);
+		padding-right: calc(var(--wp--preset--spacing--20) * 0.75);
+		padding-bottom: calc(var(--wp--preset--spacing--20) * 0.5);
+		padding-left: calc(var(--wp--preset--spacing--20) * 0.75);
```

Could shorthand this
```suggestion
		padding: calc(var(--wp--preset--spacing--20) * 0.5) calc(var(--wp--preset--spacing--20) * 0.75);
```

---

Yeah could, this was inherited from elsewhere. 

I think I'll just leave it as it. If we have another search bar like this, we'll probably want to move this code up to the parent anyways.